src/redux/reducer.js

```diff
@@ -83,4 +84,8 @@ export const reducer = createReducer(initialState, {
             };
         }
     },
+
+    [CURRENT_MERGES_LIST]: (state, action) => {
+        state.merges = action.merges;
```

Comment:
Now that you get full merge state from server for the whole day, I think the redux store scheme has to be refactored. Why do you keep  

merge: {
        process: null,
        date: null,
        igms: [],
    },
as it is ? Isn't it redundant with what is in state.merges?

Reply:
I use it just to store the list of merges per day this all

Reply:
And if you store all day merges, why do you keep current merge state. It is redundant.